earth_enterprise/src/scons/getversion.py

```diff
@@ -222,17 +244,36 @@ def set_long(self, value):
 
         self.long_version_string = value
 
+    def is_warning_available(self):
```

Comment:
Opinion: I'd name this something more indicative, maybe:

```suggestion
    def has_warning(self):
```

Also, I'm not sure this function is needed.  A function that returns all the warnings, and is able to return an empty result already fulfills the purpose of this function.

Reply:
Removed; returning None from get_warning_message()